testsuite/MDAnalysisTests/coordinates/test_writer_api.py

```diff
@@ -52,6 +52,9 @@ def test_ts_error(writer, tmpdir):
         fn = str(tmpdir.join('out.xtc'))
     elif writer == mda.coordinates.LAMMPS.DATAWriter:
         pytest.skip("DATAWriter requires integer atom types")
+    elif writer == mda.coordinates.H5MD.H5MDWriter:
+        pytest.skip("H5MDWriter requires h5py to be installed, "
+                    "fails MIN dependency test")
```

Comment:
You are _always_ excluding this test, even if hdf5 is installed. The API tests should run in full tests.

Only skip the tests if you cannot import hdf5.